protos/search.proto

```diff
@@ -32,7 +26,10 @@ message TimeSpan {
 }
 
 message Filters {
-    // Scope e.g. "confluence", "slack://channel-name", "google-drive://CompanyDrive/document.docx"
+    // Scope of the query. This may either be the name of a fetcher, or a subspace of documents.
```

Comment:
nit: perhaps the term `fetcher` doesn't have enough meaning to a user, I see `provider` was used in the readme snippet copy of this